back/engines/commercial/insights/app/services/insights/category_suggestions_service.rb

```diff
@@ -77,21 +77,32 @@ def cancel(tasks)
       tasks.each(&:destroy)
     end
 
-    # @return [Array<Hash>]
-    def documents_from(inputs)
-      inputs.map { |i| { text: input_to_text(i), doc_id: i.id } }
-            .select { |document| document[:text] }
+    private
+
+    # @param [Idea] input
+    # @return [Hash{Symbol => String}]
+    def input_to_document(input)
+      { text: input_to_text(input), doc_id: input.id }
     end
 
     # @param [Idea] input
-    # @return [String,NilClass]
+    # @return [String]
     def input_to_text(input)
-      # [TODO] We currently assume that the multiloc contains only one body.
+      # We currently assume that the multiloc contains only one locale.
       # It seems to be true in most cases. (The UI does not allow a user to provide translations.)
-      # Might not be true in seed data or similar settings.
-      # [TODO] We are not taking the title into account at the moment.
-      text = input.body_multiloc.compact.values.first
-      ActionView::Base.full_sanitizer.sanitize(text)&.strip.presence
+      title = html_to_text(input.title_multiloc.compact.values.first).strip
```

Comment:
Why do you always get first value? Is it always english?

Reply:
It's just because the hash has (supposedly) only 1 key-value pair :)

Reply:
Indeed, it doesn't seem to be possible to enter an initiative title in multiple languages. But still, the title can be in any language (depending on the current locale). Doesn't NLP care if we send French, English, Dutch the same way?

Reply:
Not in this case. This classification model is multi-lingual 🙂

Reply:
Got it, thanks. Just in case we start to store multiple values in `title_multiloc`, maybe it makes sense to iterate and join all values not only the first one. It won't do any harm, but anyway it's up to you.